src/asciidoc/docs/develop/publishLocal.adoc

```diff
@@ -247,11 +235,10 @@ TIP: When you publish your project, the Koji editor automatically creates a new
 
```

Comment:
This entire section needs a complete rewrite. There aren't any customization files anymore. "Visual mode" isn't a thing (we have previews now). VCCs are deprecated. Instead, we should be talking about how to use the debugger in conjunction with local development to test things like remix mode or in-app purchases. For some tips and tricks, you can talk to Marko. Most of this section probably just needs to be deleted.